cogs/greetings.py

```diff
@@ -3,22 +3,30 @@
 import os
 from discord.ext import commands
 
+# Class dedicated to greeting new members when joining the server
 class Greetings(commands.Cog):
+
+# Constructor that assigns the welcome message to the specific new member with their username
     def __init__(self, client):
         self.client = client
         self.welcome_message_id = self.load_welcome_message_id()
 
+# Helper function responsible for loading and returning the welcome message
     def load_welcome_message_id(self):
         try:
             with open("txtfiles/welcome_message_id.txt", "r") as file:
                 return int(file.read())
         except FileNotFoundError:
             return None
 
+# Helper function responsible for retaining the welcome message id specific to the user.
+# As well as storing the id to be loaded later if the bot goes offline in the future.
```

Comment:
This function doesn't really retain the welcome message id specific to the user. It stores the id of the welcome message (which is just sent to a general channel) by writing it to a file so that if the bot goes offline in the future, it can be reloaded without having to create a new welcome message.